### PR TITLE
e2e

### PR DESCRIPTION
**Pull request by derHowie**
This is a surrogate PR for #1702 without the skipped tests and some small e2e tweaks.

## What Changed
- Updated packages to resolve Ledger issues with chromedriver
- Bump chrome driver to 128 on CI
- Return data in gas hooks to prevent react-query error in the console after each swap input value changes

## PoW
Screenshot 2024-09-18 at 3 21 16 PM

## What to Test
- QA should run regression on Ledger flow to verify there are no issues with the dependency updates.
- Attempt to run the e2e suite locally with chrome version 128 or above and ensure that the tests run correctly, pass, and that there are no additional issues with local e2e.

### CODE REVIEW

**Review by BrodyHughes — APPROVED** [review]
👍

**Review by brunobar79 — APPROVED** [review]
🐐

File: package.json [hunk 1]
```diff
@@ -96,8 +96,10 @@
     "@ethersproject/transactions": "5.7.0",
     "@ethersproject/units": "5.7.0",
     "@ethersproject/wallet": "5.7.0",
-    "@ledgerhq/hw-app-eth": "6.35.3",
+    "@ledgerhq/evm-tools": "1.2.0",
+    "@ledgerhq/hw-app-eth": "6.38.0",
     "@ledgerhq/hw-transport-webhid": "6.28.2",
```
Comment (DanielSinclair) [inline package.json:101]: @BrodyHughes Is there a reason we dropped hw-transport-webhid to `6.28`? 6.29 is compatible with the Flex devices

Comment (BrodyHughes) [reply package.json:101]: no reason no. i don't remember specifying 28 when i bumped. i imagine bumping to 29 is fine.

Comment (BrodyHughes) [reply package.json:101]: hopefully this fixes e2e and bumping is easy now

**Pull request merged**